wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationSystemMessageTests.swift

```diff
@@ -285,23 +285,106 @@ final class ConversationSystemMessageTests: ZMSnapshotTestCase {
         verify(message: message)
     }
 
-    func testPotentialGap_addedUsers() {
+    func testPotentialGap_addedUser() {
         let message = MockMessageFactory.systemMessage(with: .potentialGap)!
 
         message.assignMockAddedUser()
 
         verify(message: message)
     }
 
+    func testPotentialGap_addedUsers() {
+        let message = MockMessageFactory.systemMessage(with: .potentialGap)!
+
+        message.assignMockAddedUsers(users: SwiftMockLoader.mockUsers().prefix(4))
+
+        verify(message: message)
+    }
+
+    func testPotentialGap_removedUser() {
+        let message = MockMessageFactory.systemMessage(with: .potentialGap)!
+
+        message.assignMockRemovedUsers(users: SwiftMockLoader.mockUsers().prefix(1))
+
+        verify(message: message)
+    }
+
     func testPotentialGap_removedUsers() {
         let message = MockMessageFactory.systemMessage(with: .potentialGap)!
 
+        message.assignMockRemovedUsers(users: SwiftMockLoader.mockUsers().prefix(4))
+
+        verify(message: message)
+    }
+
+    func testPotentialGap_addedAndRemovedOneUser() {
+        let message = MockMessageFactory.systemMessage(with: .potentialGap)!
+
+        message.assignMockAddedUser()
+        message.assignMockRemovedUsers(users: SwiftMockLoader.mockUsers().suffix(1))
+
+        verify(message: message)
+    }
+
+    func testPotentialGap_addedOneUserAndRemovedMultipleUsers() {
+        let message = MockMessageFactory.systemMessage(with: .potentialGap)!
+
+        message.assignMockAddedUser()
+        message.assignMockRemovedUsers(users: SwiftMockLoader.mockUsers().suffix(4))
+
+        verify(message: message)
+    }
+
+    func testPotentialGap_addedMultipleUsersAndRemovedOneUser() {
+        let message = MockMessageFactory.systemMessage(with: .potentialGap)!
+
+        message.assignMockAddedUsers(users: SwiftMockLoader.mockUsers().suffix(4))
+        message.assignMockRemovedUsers(users: SwiftMockLoader.mockUsers().suffix(1))
+
+        verify(message: message)
+    }
+
+    // MARK: - Potential Gap German Translation
+
+    func testPotentialGap_DE() {
```

Comment:
Why are there separate tests for DE? The DE test plan can run the same tests as we do normally.

Reply:
@johnxnguyen You're absolutely right. We started this way with the assumption that can you change the language for the tests in code. Now that we moved to a proper test plan, I removed and cleaned things up.